김진/231205/두-정수-사이의-합.js

```diff
@@ -0,0 +1,10 @@
+function solution(a, b) {
+  let result = 0;
+  let start = a - b > 0 ? b : a;
+  let stop = a - b > 0 ? a : b;
```

Comment:
```suggestion
  const start = a - b > 0 ? b : a;
  const stop = a - b > 0 ? a : b;
```
start랑 stop은 할당될 때 이후에 값이 변하지 않으니 const로 선언해주시면 좋을 것 같아요.
const 선언과 let 선언이 코드에서 같은 결과를 낸다면 const를 사용하는 것을 권고합니다.